dateparser/date.py

```diff
@@ -154,17 +156,17 @@ def __init__(self, language, date_string, date_formats):
         self._translated_date_with_formatting = None
 
     @classmethod
-    def parse(cls, language, date_string, date_formats=None):
-        instance = cls(language, date_string, date_formats)
+    def parse(cls, language, date_string, date_formats=None, settings=None):
+        instance = cls(language, date_string, date_formats, settings)
         return instance._parse()
 
     def _parse(self):
         for parser in (
-            self._try_timestamp,
-            self._try_freshness_parser,
-            self._try_given_formats,
-            self._try_dateutil_parser,
-            self._try_hardcoded_formats,
+                self._try_timestamp,
+                self._try_freshness_parser,
+                self._try_given_formats,
+                self._try_dateutil_parser,
+                self._try_hardcoded_formats,
```

Comment:
Why making indent 8 spaces? `):` placed the way it is clearly separates condition part from body.